config/config.default.yaml

```
@@ -991,6 +996,150 @@ adjustments:
           capital_cost: 1.0
     absolute: false
 
+# docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#data
+data:
+  hotmaps_industrial_sites:
+    source: "archive"
+    version: "latest"
```

```suggestion
    source: archive
    version: latest
```
Could leave out brackets?